### DISCUSSION TITLE
Joining Google Summer of Code

### DISCUSSION THREAD

**Discussion by CBID2** [EddieHubCommunity/support#5986] [Ideas] [answered]
Hi @eddiejaoude and co! :) I was scrolling through Google Summer of Code's [blog](https://opensource.googleblog.com/2022/11/get-ready-for-google-summer-of-code-2023.html) and I noticed that they're looking for mentoring organizations for next summer. I think EddieHub  should apply. I mean, the organization has done a lot to make an inclusive open source community for people from all walks of life(having a pretty large following on YouTube helps). What do you think?

**Comment by eddiejaoude** [comment, marked as answer]
Thank you Christine for the suggestion 👍  @SaraJaoude could you look into this please and see what you think?

**Reply by CBID2** [reply]
You’re welcome @eddiejaoude! I’ve been dabbling with the idea of applying, so having EddieHub as an organization will make things a bit easier (if they accept my application of course)

**Reply by CBID2** [reply]
Hi everyone! Google Summer of Code just released their [timeline for 2023](https://developers.google.com/open-source/gsoc/timeline). Applications for mentoring organizations are due 02/07/23.
